floris/simulation/wake_velocity/jensen.py

```diff
@@ -96,9 +96,9 @@ def function(
         # define the boundary of the wake model ... y = mx + b
         m = self.we
         x = x_locations - turbine_coord.x1
-        b = turbine.rotor_radius
+        b = turbine.rotor_diameter
 
-        boundary_line = m * x + b
+        boundary_line = (m * x + b) / 2.0
```

Comment:
Doesn't this reduce the wake expansion parameter by half? The original formulation by Jensen for the wake is in terms of the wake radius.

Reply:
This change means the wake expansion parameter is the wake diameter expansion rather than the radius expansion. In the `self.we` [docstring](https://github.com/NREL/floris/blob/main/floris/simulation/wake_velocity/jensen.py#L130) we say that its relative to the radius.

Related, the default value is 0.05 whereas the original Jensen paper suggests a usual value of 0.1. I don't remember how we landed on 0.05, but if it was meant to be half of the full expansion then I think it should be 0.1.

Reply:
I think if we undid both changes above the results would be the same, this was more for the point of comparison.  The switch to 0.05 was also to be more consistent with other models

Reply:
This finally was an error on my part, just pushed up a reversion to how it was